test/packages/good_v3/manifest.yml

```diff
@@ -40,10 +40,22 @@ policy_templates:
         division: observability
         team: obs-infraobs-integrations
     configuration_links:
-      - title: Security overview
-        url: "/security/overview"
+      - title: View Agents
+        url: "/agents"
         type: next_step
-        content: "View security overview"
+        content: "Check your agents in Fleet"
+        appId: "fleet"
+        internal: true
```

Comment:
Instead of adding the `internal` setting, could we use the urls prefixed by `kbn` that are used in many other places?
```
      - title: View Agents
        url: "kbn:/app/fleet/agents"
        type: next_step
        content: "Check your agents in Fleet"
```

Reply:
I couldn't find any examples in package-spec, but maybe you refer to the Kibana dev tools?

We could use this format, it requires to do some more manipulation in kibana as the url needs to be split (it cannot be used directly by kibana this way) and it's also less explicit. However, reading your other comments I understand your concerns that this could be a bit too specific to Kibana internal way of handling urls.

Reply:
Updated the PR and the description based on this discussio.